index.html

```diff
@@ -61,7 +61,14 @@
 
 	<div id="overlay"></div>
 
-	<div id="webviews"></div>
+	<div id="webviews">
+		<div id="left" class="arrow-indicator">
```

Comment:
Can you make the id's of these something more descriptive, like "leftArrowContainer"?